source/ftrack_connect_pipeline/client/asset_manager.py

```diff
@@ -37,7 +37,7 @@ def on_host_changed(self, host_connection):
         definitions = self.host_connection.definitions.get(schema_title)
         if len(definitions) > 0:
             # Only one definition for now, we don't have a definition schema on the AM
-            self.change_definition(schema, definitions[0])
+            self.change_definition(definitions[0], schema)
```

Comment:
@lluisFtrack how about **set_definition** ? 
seems more suitable then **change_definition** (as in change from ? ... )

Reply:
Or just have a simple setter?

Reply:
Or just have a simple setter?